### PR TITLE
Core: Add misc fields in ShopProduct

### PR DESCRIPTION
**Pull request by chessbr**
Enable to set a custom name, description and short description for a product of a shop.

- API: Fix viewset queryset filter
- API: Add django_filters to enable DRF filters form

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ Unrealeased
 Core
 ~~~~
 
+- Add name, description and short_description fields to ShoProduct model
```
Comment (Pikkupomo) [inline doc/changelog.rst]: typo `ShoProduct`

File: shuup/core/basket/objects.py [hunk 2]
```diff
@@ -52,7 +52,13 @@ def cache_info(self, request):
         self.gross_weight = product.gross_weight
         self.shipping_mode = product.shipping_mode
         self.sku = product.sku
-        self.text = product.safe_translation_getter("name", any_language=True)
+        self.text = ""
+
+        shop_product = self.shop_product
```
Comment (Pikkupomo) [inline shuup/core/basket/objects.py]: useless assign imo

**Comment by tulimaki** [comment]
@gurch101 can you fix these two things for your next commit to the branch.

File: shuup/core/basket/objects.py (hunk 2 above)
Comment (chessbr) [inline shuup/core/basket/objects.py]: if you check the property above, the shop_product has the following code:
```
return self.product.get_shop_instance(self.shop)
```
It is not that expensive, but.. :)

**Pull request merged**